modules/microshift-viewing-audit-logs.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+//microshift_troubleshooting/microshift-audit-logs.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-security-context-constraints-alert-eval_{context}"]
+= Identifying pod security violations through audit logs
+
+You can identify pod security admission violations on a workload by viewing the server audit logs. The following procedure shows you how to access the audit logs and parse them to find pod security admission violation in a workload.
```

Comment:
@dhellmann Hi Doug! You you think this intro looks better?

Reply:
This change looks good. I'll let @copejon review the procedures. Thanks, @kelbrown20 !